internal/factsengine/gatherers/verifypassword.go

```diff
@@ -0,0 +1,94 @@
+package gatherers
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	crypt "github.com/tredoe/osutil/user/crypt"
+	sha512crypt "github.com/tredoe/osutil/user/crypt/sha512_crypt"
+	"github.com/trento-project/agent/pkg/factsengine/entities"
+	"github.com/trento-project/agent/pkg/utils"
+)
+
+const (
+	VerifyPasswordGathererName = "verify_password"
+	checkableUsernames         = "hacluster"
+	unsafePasswords            = "linux"
```

Comment:
why are we splitting by comma instead of using a slice here?

Reply:
I was trying to avoid not using a const but you are totally right with your comment below, changing it.